docs/reference/docs/reindex.asciidoc

```diff
@@ -1119,6 +1122,30 @@ POST _reindex
 // TEST[s/otherhost:9200/\${host}/]
 // TEST[s/\.\.\.,/"username": "test_admin", "password": "x-pack-test-password",/]
 
+[[ccs-based-reindex-from-remote]]
+==== ccs based reindex from remote
+
+Reindex supports Cross-cluster search (<<modules-cross-cluster-search,{ccs}>>) based reindexing from a remote Elasticsearch cluster:
```

Comment:
```suggestion

[[reindex-from-remote-cluster]]
==== reindex from remote cluster

Reindex supports sourcing documents from a remote Elasticsearch cluster:
```

Reply:
Since a section named [reindex-from-remote](https://github.com/elastic/elasticsearch/blob/2895f1e900b2f41704fd507845102a281cff437e/docs/reference/docs/reindex.asciidoc?plain=1#L958) already exists, won't it be difficult to understand what the difference is with `reindex-from-remote-cluster`?
You said that ccs is a low-level feature and we don't need to worry about it too much, but I think it would be better to clarify the difference with the existing remote reindex.